set.mm

```diff
@@ -613966,14 +613999,14 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
     sqmid3api.b $e |- ( A + N ) = B $.
     sqmid3api.c $e |- ( B + N ) = C $.
     $( Value of the square of the middle term of a 3-term arithmetic
-       progression.  (Contributed by Steven Nguyen, 20-Sep-2022.) $)
+       progression.  (Contributed by Steven Nguyen, 20-Sep-2022.)  (Proof
+       shortened by Steven Nguyen, 10-Dec-2022.) $)
```

Comment:
Since it is in a mathbox and with the same contributor, the second tag is not necessary. Up to you.